modules/core/src/core-layers/text-layer/text-layer.js

```diff
@@ -54,23 +54,32 @@ const defaultProps = {
 
 export default class TextLayer extends CompositeLayer {
   initializeState() {
-    const {gl} = this.context;
-    const {fontFamily} = this.props;
-    const {mapping, texture} = makeFontAtlas(gl, fontFamily);
-    this.state = {
-      iconAtlas: texture,
-      iconMapping: mapping
-    };
+    this.state = {};
+    this.updateFontAtlas(this.props.fontFamily);
```

Comment:
This should be removed. `oldProps.fontFamily !== props.fontFamily` is triggered on initialization.